.github/workflows/labeler.yml

```diff
@@ -1,11 +1,11 @@
+---
 name: "Pull Request Labeler"
-on:
-- pull_request_target
-
+on: pull_request_target
 jobs:
-  triage:
+  labeler:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@main
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+      - uses: actions/labeler@main
+        with:
+          sync-labels: ''
```

Comment:
i think by default `sync-labels` is `false`. see https://github.com/actions/labeler/blob/fa244eaba5c3a196c5312fbef1e5ff0cf5350dc5/src/main.ts#L17 and https://github.com/actions/toolkit/blob/0759cdc2309acfd91b6503597dcfa02a3cee99d0/packages/core/src/core.ts#L84-L92

Reply:
@tchaikov after checking that code, my bet is that the defaults from the [`action.yml`](https://github.com/actions/labeler/blob/fa244eaba5c3a196c5312fbef1e5ff0cf5350dc5/action.yml) are somehow injected in the environment, and that's why the empty string doesn't work:
```s
> !!'false'
true

> !!''
false
```

Reply:
There it is: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/docs/github-package.md#mocking-inputs